lib/widgets/user_name.dart

```diff
@@ -5,14 +5,15 @@ import 'package:provider/provider.dart';
 
 class UserName extends StatelessWidget {
   final String login;
+  final String prefix;
 
-  UserName(this.login);
+  UserName(this.login, this.prefix);
 
   @override
   Widget build(BuildContext context) {
     final theme = Provider.of<ThemeModel>(context);
     return Link(
-      url: '/github/$login',
+      url: '/$prefix/$login',
```

Comment:
Perhaps we could make this more general, because some platforms like GitLab use `id` instead of `login`.

How about replacing the `String prefix` with `String userLink`?

Reply:
Makes sense but in the same file [this line](https://github.com/git-touch/git-touch/blob/4679ca10ee1898a10798c10091445f09bfa26617/lib/widgets/user_name.dart#L22) uses `login`. But I also noticed that `github` and `gitee` have URLs like this: `/github/login` while `gitlab` has like this: `/gitlab/user/id`. 
So maybe `userLink` could be the part before `login` and we pass in `id` as login for `gitlab`? What do you think?

Reply:
Oh, I missed that. This widget seems a little over-encapsulation.

Let's keep it as is